dts/bindings/power-domain/ti,cc13xx-cc26xx-power-domain.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI SimpleLink CC13xx / CC26xx power domain
+
+compatible: "ti,cc13xx-cc26xx-power-domain"
+
+include: power-domain.yaml
+
+properties:
+    resource-id:
+      type: int
+      required: true
+      description:
+        Power domain resource ID as defined in prcm.h (TI HAL).
```

Comment:
would be nice to provide a dt-bindings header with pre-defined resource IDs (DT can't include HAL headers).

Reply:
something like this: https://github.com/ceolin/zephyr/blob/pm/pd/dts/bindings/spi/xlnx,xps-spi-2.00.a.yaml#L20-L29 ?

Not sure if this helps here, since an integer value will also not being that meaningful. Also, it is pretty common set value directly in DT, all registers are arbitrarily set.

Reply:
I mean to provide a header in `include/dts-bindings/` with pre-defined resource-ids.

So:
```
			/* PRCM_DOMAIN_PERIPH value */
			status = "okay";
			resource-id = <0x00000004>;
```

can be written as:
```
			status = "okay";
			resource-id = <PRCM_DOMAIN_PERIPH>;
```